<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="QLabelVehicleInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>车辆信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelGear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前档位</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelWheelAngle">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>方向盘角度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditGear">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditWheelAngle">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelSpeed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>车辆速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelAccPedal">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>油门踏板</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditCarSpeed">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditAccPedal">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelEPBState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EPB状态</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditEPBState">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelVehicleInfo_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>定位姿态</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditGaussX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditGPSStatus">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelGaussY">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GaussY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditGaussY">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelGaussX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GaussX</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditAzimuth">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelRoll">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横滚角</string>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelAzimuth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航向角</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditRoll">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelGPSStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>GPS状态</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditPitch">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="QLabelPitch">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>俯仰角</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
